### PR TITLE
Fix GetGameSoundParams CSGO

### PR DESCRIPTION
**Pull request by komashchenko**
In CSGO engine, cannot pass index directly to `handle` parameter
SDK update required https://github.com/alliedmodders/hl2sdk/pull/97

### CODE REVIEW

**Comment by Wend4r** [comment]
Works
```
sm_test_game_sound SprayCan.PaintGhost
Game sound "SprayCan.PaintGhost": channel = 6, soundLevel = 75, volume = 0.899902, pitch = 100, sample = "~)player/halloween/spray_01.wav"
sm_test_game_sound SprayCan.PaintGhost123
Game sound "SprayCan.PaintGhost123" is not found
```
The game sound (for example, `SprayCan.PaintGhost`) is being played
image

[TestGetGameSoundParams.zip](https://github.com/alliedmodders/sourcemod/files/7477477/TestGetGameSoundParams.zip)

File: extensions/sdktools/vsound.cpp [hunk 1]
```diff
@@ -648,13 +648,20 @@ bool GetSoundParams(CSoundParameters *soundParams, const char *soundname, cell_t
 	if ( !soundname[0] )
 		return false;
 
+#if SOURCE_ENGINE == SE_CSGO || SOURCE_ENGINE == SE_BLADE
+	HSOUNDSCRIPTHASH index = soundemitterbase->HashSoundName(soundname);
+	
+	if(!soundemitterbase->IsValidHash(index))
+		return false;
+#else
```
Comment (KyleSanderson) [inline extensions/sdktools/vsound.cpp:656]: These need comments without otherwise indentation.

**Review by KyleSanderson — COMMENTED** [review]
Just comments I think.

**Comment by peace-maker** [comment]
Can you elaborate quickly why you disabled precaching please?

**Review by peace-maker — APPROVED** [review]
Thanks!

**Pull request merged**